analyses/snv-callers/README.md

```diff
@@ -1,213 +1,75 @@
 # SNV caller comparison analysis
 
 This analysis evaluates [MAF files](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) from different SNV callers, compares their output, and creates a [consensus mutation file](./results/consensus/consensus_mutation.maf.tsv.zip).
-This consensus mutation file is [MAF-like](#consensus-mutation-call) meaning it is TSV file that contains many of the fields of a [MAF file](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) but also some added calculations like [Variant Allele Fraction](#variant-allele-fraction-calculation) and some sample metadata information.
-
-**Table of Contents**
-* [How to run this pipeline](#how-to-run-this-pipeline)
-* [General usage of scripts](#general-usage-of-scripts)
-* [Individual caller evaluation](#individual-caller-evaluation)  
-  * [Base change analysis](#base-change-analysis)  
-  * [Variant allele fraction calculation](#variant-allele-fraction-calculation)  
-  * [Genomic regional analyses](#genomic-regional-analyses)  
-  * [Tumor mutation burden](#tumor-mutation-burden-calculation)
-  * [COSMIC mutation overlap](#cosmic-mutation-overlap)  
-* [Comparison analysis](#comparison-of-callers)
- * [Mutation IDs](#mutation-ids)
- * [Consensus mutation calls](#consensus-mutation-calls)
-* [Overall file structure](#overall-file-structure)
-* [Summary of functions](#summary-of-custom-functions)
+This consensus mutation file is [MAF-like](#consensus-mutation-call) meaning it is TSV file that contains many of the fields of a [MAF file](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) but also has [VAF](#variant-allele-fraction-calculation)
```

Comment:
I am pretty sure we are including the full set of fields now, as they are all in the database. 
```suggestion
This consensus mutation file is [MAF-like](#consensus-mutation-call) meaning it is TSV file that contains the fields of a [MAF file](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) but adds [VAF](#variant-allele-fraction-calculation), but does not contain a starting comment line with a version number.
```